src/lib/template-result.ts

```diff
@@ -100,7 +121,7 @@ export class TemplateResult {
 
   getTemplateElement(): HTMLTemplateElement {
     const template = document.createElement('template');
-    template.innerHTML = this.getHTML();
+    template.innerHTML = dangerouslyTurnToTrustedHTML(this.getHTML()) as string;
```

Comment:
The reason this is secure is because `this.strings` is a TemplateStringsArray. Add a TODO to use `Array.isTemplateObject` to validate `this.strings` if and when it is implemented, and maybe link to https://github.com/tc39/proposal-array-is-template-object